addons/google_spreadsheet/static/src/add_to_google_spreadsheet/add_to_google_spreadsheet.js

```diff
@@ -5,6 +5,7 @@ import { useService } from "@web/core/utils/hooks";
 
 const { Component } = owl;
 const favoriteMenuRegistry = registry.category("favoriteMenu");
+const Domain = require('web.Domain');
```

Comment:
You should probably use `import` to prevent the failing `ci/runbot` check.

```suggestion
import Domain from "web.Domain";
```

Otherwise, LGTM